src/bufr/BufrParser/BufrDescription.cpp

```diff
@@ -32,15 +31,6 @@ namespace Ingester
     {
         setFilepath(conf.getString(ConfKeys::Filename));
 
-        if (conf.has(ConfKeys::IsWmoFormat) && conf.getBool(ConfKeys::IsWmoFormat))
```

Comment:
Just for my curiousity, why are the WMO format changes included in this PR? Is it required for this API?